### PR TITLE
Proof reading of README.md

### PR DESCRIPTION
**Pull request by Hrubon**
@dcepelik, please check. If you see any other typos, please shout. Thank you!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -152,13 +153,15 @@ The following rules apply to the slice parsing:
   their underlying data-type.
 * The items are separated by comma. If one needs a comma to be present in a
   slice item, it must be escaped by back-slash, like this: `\,`. The same
-  applies to the back-slash itself: `\\`.
-* A double-quotes are also reserved for special use. All commas and
```
Comment (dcepelik) [inline README.md:156]: A double -> double?

File: README.md [hunk 2]
```diff
@@ -152,13 +153,15 @@ The following rules apply to the slice parsing:
...
-  back-slashes inside double quotes are escaped automatically. If one needs
-  a double-quote to be present in a slice item, it must be **always**
-  escaped by back-slash like this: `\"`.
+  applies to the back-slash itself: `\\`. Generally, all characters prefixed
+  by back-slash are expanded to the respective character after the
+  back-slash.
+* A double-quotes are also reserved for special use. All commas inside
```
Comment (dcepelik) [inline README.md:159]: A double quotes -> double quotes?

File: README.md [hunk 3]
```diff
@@ -152,13 +153,15 @@ The following rules apply to the slice parsing:
...
+  double quotes are escaped automatically. If one needs a double-quote to be
+  present in a slice item, it must be **always** escaped by back-slash like
+  this: `\"`.
 * All leading and trailing spaces at the item boundaries (before and after
-  comma), are ignored. Spaces inside double-quotes are never ignored.
+  comma) are ignored. Spaces inside double-quotes are never ignored.
```
Comment (dcepelik) [inline README.md:164]: Nitpick: not just after/before comma but also at the beginning and the end (I think it works like that?).

File: README.md [hunk 4]
```diff
@@ -152,13 +153,15 @@ The following rules apply to the slice parsing:
...
-  applies to the back-slash itself: `\\`.
-* A double-quotes are also reserved for special use. All commas and
-  back-slashes inside double quotes are escaped automatically. If one needs
-  a double-quote to be present in a slice item, it must be **always**
-  escaped by back-slash like this: `\"`.
+  applies to the back-slash itself: `\\`. Generally, all characters prefixed
+  by back-slash are expanded to the respective character after the
+  back-slash.
+* A double-quotes are also reserved for special use. All commas inside
+  double quotes are escaped automatically. If one needs a double-quote to be
```
Comment (dcepelik) [inline README.md:160]: > All commas inside double quotes are escaped automatically

Sounds like voodoo. How about

> Commas in double-quoted fields have no special meaning.

File: README.md [hunk 5]
```diff
@@ -4,8 +4,9 @@
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
```
Comment (dcepelik) [inline README.md:4]: Thank you, the README is definitely improved.

**Review by dcepelik — CHANGES_REQUESTED** [review]
Minor changes requested.

File: README.md (hunk 3 above)
Comment (Hrubon) [reply README.md:164]: True. :)

File: README.md (hunk 5 above)
Comment (Hrubon) [reply README.md:4]: NP. :+1:

**Pull request merged**